### DISCUSSION TITLE
[FEATURE REQUEST] Database to store sensitive data (passwords and other)

### DISCUSSION THREAD

**Discussion by N1coc4colA** [linuxdeepin/developer-center#2936] [Features Request & Ideas | 特性请求 & 头脑风暴]
## KWallet, Gnome Keyring or like

Hello, I'm building an app and now I see that storing a password is complicated, and for any sensitive data... So is it possible to provide a tool like Gnome Keyring or KWallet (which should be better)? For example, to save passwords of Chromium there instead of the app itself. It can be usefull to provide a generalised way to use passwords or store those things. I tried KWallet implementation in my app and it is very easy to use, I didn't took a look at Gnome Keyring and its API. I hope this feature will be implemented as it is simple to make it on top of KWallet. Maybe use the Deepin Cloud password for that and be able to add custom wallets...

@n1coc4cola

**Comment by meiyixiang** [comment]
Hello, currently you can use Seahorse
